webpack.common.js

```
@@ -0,0 +1,80 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
+  template : 'index.html',
+  filename : 'index.html',
+  inject : 'body',
+
+});
+const outputSubDir = 'src/main/resources/static'
+
+var config = {
+  context: __dirname + '/src/frontend',
```

ESLint

[error] Use path.join() or path.resolve() instead of + to create paths.

(no-path-concat)